runtime/centrifuge/src/lib.rs

```diff
@@ -1000,6 +1080,410 @@ impl pallet_collator_selection::Config for Runtime {
+				who,
+				Role::PoolRole(PoolRole::TrancheInvestor(
+					tranche.of_tranche(),
+					T::now().as_secs(),
+				)),
+			),
+		};
+
+		if is_tranche_investor {
+			Ok(())
+		} else {
+			// TODO: We should adapt the permissions pallets interface to return an error instead of a boolen. This makes the redundant has not role error
+			//       that downstream pallets always need to generate not needed anymore.
+			Err(DispatchError::Other(
+				"Account does not have the TrancheInvestor permission.",
+			))
+		}
+	}
+}
+
+impl pallet_investments::Config for Runtime {
+	type Accountant = PoolSystem;
+	type Amount = Balance;
+	type BalanceRatio = Rate;
+	type InvestmentId = TrancheCurrency;
+	type MaxOutstandingCollects = MaxOutstandingCollects;
+	type PreConditions = IsTrancheInvestor<Permissions, Timestamp>;
+	type RuntimeEvent = RuntimeEvent;
+	type Tokens = Tokens;
+	type WeightInfo = ();
+}
+
+impl pallet_interest_accrual::Config for Runtime {
+	type Balance = Balance;
+	type InterestRate = Rate;
+	// TODO: This is a stopgap value until we can calculate it correctly with updated benchmarks. See #1024
+	type MaxRateCount = MaxActiveLoansPerPool;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	type Weights = ();
+}
+
+parameter_types! {
+	pub const LoansPalletId: PalletId = cfg_types::ids::LOANS_PALLET_ID;
+	pub const MaxActiveLoansPerPool: u32 = 50;
+	pub const MaxWriteOffGroups: u32 = 100;
+}
+
+impl pallet_loans::Config for Runtime {
+	type Balance = Balance;
+	type BlockNumberProvider = System;
+	type ClassId = CollectionId;
+	type CurrencyId = CurrencyId;
+	type InterestAccrual = InterestAccrual;
+	type LoanId = ItemId;
+	type LoansPalletId = LoansPalletId;
+	type MaxActiveLoansPerPool = MaxActiveLoansPerPool;
+	type MaxWriteOffGroups = MaxWriteOffGroups;
+	type NonFungible = Uniques;
+	type Permission = Permissions;
+	type Pool = PoolSystem;
+	type Rate = Rate;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	type WeightInfo = pallet_loans::weights::SubstrateWeight<Self>;
+}
+
+parameter_types! {
+	pub const MaxKeys: u32 = 10;
+	pub const DefaultKeyDeposit: Balance = 100 * CFG;
+}
+
+impl pallet_keystore::pallet::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<AllOfCouncil>;
+	type Balance = Balance;
+	type Currency = Balances;
+	type DefaultKeyDeposit = DefaultKeyDeposit;
+	type MaxKeys = MaxKeys;
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = pallet_keystore::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+	// per byte deposit is 0.01 AIR
+	pub const DepositPerByte: Balance = CENTI_CFG;
+	// Base deposit to add attribute is 0.1 AIR
+	pub const AttributeDepositBase: Balance = 10 * CENTI_CFG;
+	// Base deposit to add metadata is 0.1 AIR
+	pub const MetadataDepositBase: Balance = 10 * CENTI_CFG;
+	// Deposit to create a class is 1 AIR
+	pub const CollectionDeposit: Balance = CFG;
+	// Deposit to create a class is 0.1 AIR
+	pub const ItemDeposit: Balance = 10 * CENTI_CFG;
```

Comment:
@mustermeiszer @offerijns These are copied 1:1 from Altair, but these deposit values feel pretty low to me. My gut says we should set them 10x-100x, but I have absolutely nothing scientific to back that up with.

Reply:
I agree. But not sure what we should base them on economically. Usually something I would ask Cassidy.